src/app/data-model/FabricScoped.h

```diff
@@ -28,21 +28,21 @@ namespace DataModel {
 /*
  * Check whether a cluster object struct is fabric scoped.
  *   A fabric scoped struct contains a field of "FabricIndex" type, however, we cannot tell the difference between that field and
- * other uint8_t fields. Thus we add a MatchesFabricIndex member function for checking the fabric id. Here, IsFabricScoped check the
- * presence of MatchesFabricIndex function. This template can be used with std::enable_if.
+ * other uint8_t fields. Thus we add a GetFabricIndex member function for checking the fabric id. Here, IsFabricScoped check the
```

Comment:
This is not about checking the fabric id anymore, right?  It's about checking the FabricIndex....

Reply:
Thanks